en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+organization ID. Each proxy and target error category result set contains breakdown of errors as 4xx, 5xx and total.
+Optionally, you can configure the `errorsByStatusCodeFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorsByStatusCodeFilter</td>
+<td valign="top"><a href="#errorsbystatuscodefilter">ErrorsByStatusCodeFilter</a></td>
+<td>
+
+Errors by status code filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getErrorsByStatusCode</strong></td>
+<td valign="top"><a href="#errorsbystatuscode">ErrorsByStatusCode</a></td>
+<td>
+
+Returns hit count for each error response code (401, 404, 500, etc.) for each API within the specified time range
+for the given combination of tenant ID, environment ID, and organization ID.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorCountByStatusCodeFilter</td>
+<td valign="top"><a href="#errorcountbystatuscodefilter">ErrorCountByStatusCodeFilter</a></td>
+<td>
+
+Error count by status code filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getTotalErrorsByAPI</strong></td>
+<td valign="top"><a href="#totalerror">TotalError</a></td>
+<td>
+
+Returns the total proxy error hit count for an API across all the data available time range for the given
+combination of tenant ID, environment ID, and organization ID. Optionally, you can configure the `filter` parameter
+to further filter the results by a time range.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">filter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a></td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">apiId</td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+API Id
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getAPIUsageOverTime</strong></td>
+<td valign="top">[<a href="#apiusageovertime">APIUsageOverTime</a>]</td>
+<td>
+
+Returns a list of each API usage over time for each time interval (calculates according to the granularity value)
```

Comment:
Change "calculates" to "calculated".

In the first phrase, do you mean "Returns the usage details for each API"?

"over time for each time interval (calculates according to the granularity value) within the specified time range" is too confusing.

Consider rephrasing.